package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "action-tailscale",
```

Comment:
Should do a sweep through here to update metadata (e.g., name, version, description, repository, author, license, etc.)